src/example.ts

```diff
@@ -136,8 +144,11 @@ const samlConfig: SamlConfig = {
   validateInResponseTo: true,
 };
 
-const acs: AssertionConsumerServiceT = async (payload) => {
-  logger.info("acs:%s", JSON.stringify(payload));
+const acs: AssertionConsumerServiceT<ExtraParamsT> = async (
+  payload,
+  extraParams
+) => {
+  logger.info("acs:%s%s", JSON.stringify(payload), JSON.stringify(extraParams));
```

Comment:
This log is needed or was added only to test the implementation?

Reply:
It was already there, I just added a second parameter.. Anyway, it's just an example.